MusicHub/users/views/profile_view.py

```diff
@@ -0,0 +1,79 @@
+from django.core.exceptions import ValidationError
+from django.utils.datastructures import MultiValueDictKeyError
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.status import *
+
+from MusicHub.main.exception_handler import (
+    CustomUserException,
+    custom_exception_handler,
+)
+from MusicHub.users import custom_user_schema
+from MusicHub.users.models import User
+from MusicHub.users.serializers import ChangePasswordSerializer, ProfileSerializer
+
+
+class ProfileView(GenericAPIView):
```

Comment:
I remember that you didn't use more specific views because problems with swagger but we need to resolve it in order to be free use DRF

Reply:
updated view using a specific view.